modelon/impact/client/entities.py

```diff
@@ -819,8 +829,18 @@ def new_experiment_definition(
                 The FMI version. Valid options are '1.0' and '2.0'. Default: '2.0'.
 
             platform --
-                Platform for FMU binary. Supported values are "auto", "win64", "win32"
-                or "linux64". Default: 'auto'.
+                Platform for FMU binary.The OS running the Impact server must match the
```

Comment:
@efredriksson-modelon , There was a bit of confusion around the platform argument sometime back. --> https://www.yammer.com/modelon.com/#/Threads/show?threadId=960989567983616. I have added the disclaimer we have in QuickStart in the function docs too.